lib/vast/CodeGen/DefaultTypeVisitor.cpp

```diff
@@ -272,6 +281,20 @@ namespace vast::cg {
         ).freeze();
     }
 
+    mlir_type default_type_visitor::VisitVectorType(const clang::VectorType *ty) {
+        return VisitVectorType(ty, clang_qualifiers());
+    }
+
+    mlir_type default_type_visitor::VisitVectorType(const clang::VectorType *ty, clang_qualifiers quals) {
+        return with_cvr_qualifiers(
+            compose_type< hl::VectorType >()
+                // bind also uninitialized size
+                .bind_always(get_size_attr(ty, self.mcontext()))
```

Comment:
This is always unitialized, or I don't get the comment.

`get_size_attr` always creates initialized `hl::SizeParam`

Reply:
This is a mistake, this comment should not be here